### DISCUSSION TITLE
Calculation surface areas not comprehensible

### DISCUSSION THREAD

**Discussion by davbus** [Open-Systems-Pharmacology/Forum#764] [Questions & Problems] [answered]
Hi,

To calculate flows from the interstitial to intracellular space (and vice versa) the permeability-surface area product is employed. The documentation for the prediction of the surface area (interstitial/intracellular) in MoBi states for e.g. fat tissue: 

SA [cm2] = (V_fat * 1000 / 14.2)^0.75 * 5 * 1E2. 

V_fat is the intersitial volume in fat tissue according to the reference, but the exponent 0.75 would not yield cm2 as a unit. Would you ellaborate on the provided equation and if possible point at where to find further information on the calculation of cellular surface areas in PK-Sim? I am aware that the information must be available somewhere in MoBi but could not find it yet.

Thank you very much in advance!
Best wishes,
David

**Comment by msevestre** [comment]
@davbus I'd suggest asking your question in the forum. I'll move this issue there on your behalf

**Comment by StephanSchaller** [comment]
Hi @davbus , this formula is set up not in an ideal way (hardcoded values) and thus may need some explanation. It is a fitted/informed allometric function and its constants can be interpreted the following way:

- the 14.2 are a fitted reference value for fat volume (in "theoretical dimension" volume, e.g. cm^3 and the dimension with "v_fat" cancel out)
- the 5*1e2 are the fitted reference value for fat surface area (in "theoretical dimension" surface, e.g., cm^2)

So in the end the formula adds up to the surface dimension, but the way it is set up technically (using the dimensionless constants), it  does not. So it is a technicality.

(This answer might need confirmation by @Yuri05 ;-))

**Reply by jessechao830330** [reply]
Hello @StephanSchaller ,

I am trying to double-check the value Pk-Sim gave me. However, I find that I couldn't get the exact same number that Pk-SIm gave me on the SA_int/cell.
In addition, the difference is more than 1-fold different between my calculation and the software value.
Take @davbus previous mention for example.
V_fat is the interstitial volume in fat tissue according to the reference, in other words, V_fat = Fat|Organ|volume * Fat|Fraction Interstitial.
when I input this calculated V_fat (0.1285) into the equation SA [cm2] = (V_fat * 1000 / 14.2)^0.75 * 5 * 1E2.
My calculated SA is 2.6087e+03, but the software gave me 6587.05365067142 instead.

What could be the reason for this difference to occur?

Please provide me with some clues or correct me. Thank you very much.

<img width="1680" alt="Screenshot 2022-09-24 at 17 09 06" src="https://user-images.githubusercontent.com/66793768/192105708-9b978ebe-fa2b-4967-ac5c-84c315f56a4d.png">

**Reply by Yuri05** [reply]
>V_fat is the interstitial volume in fat tissue according to the reference, in other words, V_fat = Fat|Organ|volume * Fat|Fraction Interstitial.

V_fat is not the interstitial volume but the whole organ volume (note  `".."` in the parameter reference - meaning that the **parent container of Fat|Interstitial** is used - which is **Fat** (whole organ))

**Reply by jessechao830330** [reply]
> > V_fat is the interstitial volume in fat tissue according to the reference, in other words, V_fat = Fat|Organ|volume * Fat|Fraction Interstitial.
> 
> V_fat is not the interstitial volume but the whole organ volume (note `".."` in the parameter reference - meaning that the **parent container of Fat|Interstitial** is used - which is **Fat** (whole organ))

Hello @Yuri05,

Thank you for your reply. However, even if I used the whole organ volume which is 0.9519 in this case with the calculation (0.9519*1000/14.2)^0.75*5*100 the answer is 1.1714e+04 which is still not matched with the value showed above? What could be the reason? or if I made any other mistake?
Thank you.

**Reply by Yuri05** [reply]
All calculations are performed in the **internal units** (https://docs.open-systems-pharmacology.org/appendix/appendix#a.1.-dimensions-and-base-units)
So if you want to calculate the value on your own: 
* you must convert the fat volume into the internal unit of the dimension "**Volume**" (which is `[L]`).
* The formula will then return the value again in the internal unit of the dimension "**Area**" (which is `[dm²]`). 
* Now you can convert the returned value from `[dm²]` to `[cm²]` and compare it with the displayed value - should be the same.

**Comment by davbus** [comment]
Hi @StephanSchaller, 
Many thanks for your reply! That makes perfect sense to me. I guess these values were determined as presented in the supplementary of this publication: [10.1002/psp4.12205](url). Are you aware if this is published for all organs represented in PK-Sim for further reading on the topic or is this Bayer internal work that was never shared (I mean beyond PK-Sim of course)? 
Best wishes,
David

**Comment by StephanSchaller** [comment]
Actually, I don't know any publication from the top of my head. @Yuri05 might be able to help.

**Comment by JanSchlender** [comment]
A recent publication on the large molecule model framework explains the concept as well (although not touching on the small molecule parametrization). Maybe this provides some clarity. Otherwise @Christoph27 might help here
https://link.springer.com/article/10.1007/s10928-017-9559-4

**Comment by Christoph27** [comment, marked as answer]
Sorry for my late answer (was on holidays). I also don't have any better reference than the supplemental material in 10.1002/psp4.12205 you mentioned. Since your question is on the interstitial/intracellular surface area, you won't find anything on that in the paper Jan mentioned. There only the plasma/interstitial surface area is given which is estimated by a completely different rationale (the proteins are not assumed to diffuse into cells).
